javaparser-core/src/main/java/com/github/javaparser/resolution/Context.java

```diff
@@ -42,21 +40,18 @@
  * @author Federico Tomassetti
  */
 public interface Context {
-	
-	/**
-	 * Returns the node wrapped in the context
-	 * 
-	 */
-	<N extends Node> N getWrappedNode();
+
+    /**
+     * Returns the node wrapped in the context
+     */
+    <N extends Node> N getWrappedNode();
 
     /**
      * @return The parent context, if there is one. For example, a method exists within a compilation unit.
      */
     Optional<Context> getParent();
 
-
```

Comment:
Can you adapt the project configuration in your IDE so as not to multiply the code modifications?

Reply:
I'm using the included `dev-files/JavaParser-idea.xml` for my project configuration, so my IDE formatting should be up to project standards. I suspect this is a change from code generator output. When I run the code generators, it effectively rewrites every source file, even if there isn't anything new being added, resulting in a lot of style changes that aren't relevant to anything being done. I've been cherry-picking files that are actually relevant, so with this being new, it wouldn't have been included before.

The proper fix is to set up automatic formatting and run it as a commit hook, which would avoid any unnecessary style changes, but I'm not sure what the best way is to do that, both in which tools to use and how to minimise impact on open PRs. Until some kind of automatic formatting exists, I think manually removing generated formatting changes from PRs is more effort than it's worth, especially since these would have to be repeated for every new codegen PR.